src/Bundle/Renderer/TwigGridRenderer.php

```diff
@@ -113,14 +113,14 @@ public function renderAction(GridViewInterface $gridView, Action $action, $data
     public function renderFilter(GridViewInterface $gridView, Filter $filter)
     {
         $template = $this->getFilterTemplate($filter);
+        $filterName = str_replace('\\', '', $filter->getName());
 
         $form = $this->formFactory->createNamed('criteria', FormType::class, [], [
-            'allow_extra_fields' => true,
```

Comment:
I don't understand why this option was here.
In the test application, it breaks with this following error (but it works on all of our projects and I don't understand why).
![Capture d’écran du 2025-04-11 11-59-11](https://github.com/user-attachments/assets/bc496306-f778-41cd-b258-52b829a84483)